client-course-schedulizer/src/components/reuseables/Schedule/Schedule.tsx

```diff
@@ -119,7 +119,7 @@ const ScheduleBase = ({ calendarHeaders, groupedEvents, ...calendarOptions }: Sc
           horizontal: "left",
           vertical: "bottom",
         }}
-        PaperProps={{ style: { maxWidth: "50%", minWidth: "500px" } }}
+        PaperProps={{ style: { maxHeight: "95%", maxWidth: "90%", minWidth: "500px" } }}
```

Comment:
With 95%, it still gets cut off. Sorry that there is such weird behavior with MacOS. 90% should work. Thanks for the fixes.  

```suggestion
        PaperProps={{ style: { maxHeight: "90%", maxWidth: "90%", minWidth: "500px" } }}
```